src/routes/(marketing)/marketing-layout.css

```
@@ -0,0 +1,252 @@
+	--m-accent-border: color-mix(in oklch, var(--primary) 40%, transparent);
+	--m-accent-line: color-mix(in oklch, var(--primary) 60%, transparent);
+	--m-green: oklch(0.65 0.22 155);
+	--m-green-dim: oklch(0.65 0.22 155 / 15%);
+	--m-green-soft: oklch(0.65 0.22 155 / 7%);
+	--m-green-border: oklch(0.65 0.22 155 / 20%);
+	--m-amber: oklch(0.78 0.18 75);
+	--m-amber-dim: oklch(0.78 0.18 75 / 15%);
+	--m-amber-soft: oklch(0.78 0.18 75 / 7%);
+	--m-amber-border: oklch(0.78 0.18 75 / 20%);
+	--m-text-3: color-mix(in oklch, var(--muted-foreground) 78%, var(--background) 22%);
+	background: var(--m-bg);
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)